mipac/actions/note.py

```diff
@@ -284,6 +279,37 @@ async def create_quote(
 
         return Note(res['created_note'], client=self._client)
 
+    @cache(group='translate_note')
+    async def translate(
+        self,
+        note_id: Optional[str] = None,
+        target_lang: str = 'en-US',
+    ) -> NoteTranslateResult:
+        """
+        ノートを翻訳します
+
+        Parameters
+        ----------
+        note_id : Optional[str], default=None
+            翻訳したいノートのID
+        target_lang : str, default='en'
+            翻訳先の言語
+
+        Returns
+        -------
+        NoteTranslateResult
+            翻訳結果
+        """
+        note_id = note_id or self._note_id
+
+        data = {'noteId': note_id, 'targetLang': target_lang}
+        res = await self._session.request(
```

Comment:
Please specify the return value
`res: INoteTranslateResult = await self._session.request(`